### PR TITLE
Weak Security

### PR DESCRIPTION
**Pull request by RudyLaurent**
### Status
**READY** 

### Description
Add security on the authentication and the password. You can activate both in `General configuration -> Security`
and set up the parameters.

The authentication security will block the authentication after too many attempt.
The password security will define what the password needed.

### CODE REVIEW

File: plugins/language/en_GB/en_GB.txt [hunk 1]
```diff
@@ -1362,6 +1363,19 @@
 1484 Time of validity of a CVE after OCS Inventory scan
 1485 By software
 1486 By computer
+1487 The authentication is temporarily blocked for too many failed. Please try later.
```
Comment (Lea9250) [inline plugins/language/en_GB/en_GB.txt:1366]: maybe indicate for how long the authentication will be blocked ? translation can be improved

File: plugins/language/en_GB/en_GB.txt [hunk 2]
```diff
@@ -1362,6 +1363,19 @@
...
+1488 Enable authentication security
+1489 Number of attempt
```
Comment (Lea9250) [inline plugins/language/en_GB/en_GB.txt:1368]: attempt is missing an 's'

File: plugins/language/en_GB/en_GB.txt [hunk 3]
```diff
@@ -1362,6 +1363,19 @@
...
+1490 Time to waiting
```
Comment (Lea9250) [inline plugins/language/en_GB/en_GB.txt:1369]: "cooldown" maybe ?

File: require/function_config_generale.php [hunk 4]
```diff
@@ -1002,6 +1005,31 @@ function pageswol() {
...
+    'SECURITY_PASSWORD_MIN_CHAR' => 'SECURITY_PASSWORD_MIN_CHAR', 
+    'SECURITY_PASSWORD_FORCE_NB' => 'SECURITY_PASSWORD_FORCE_NB',
+    'SECURITY_PASSWORD_FORCE_UPPER' => 'SECURITY_PASSWORD_FORCE_UPPER', 
+    'SECURITY_PASSWORD_FORCE_SPE_CHAR' => 'SECURITY_PASSWORD_FORCE_SPE_CHAR');
+
+    $values = look_config_default_values($champs);
+
+    ligne('SECURITY_AUTHENTICATION_BLOCK_IP', $l->g(1488), 'radio', array(1 => 'ON', 0 => 'OFF', 'VALUE' => $values['ivalue']['SECURITY_AUTHENTICATION_BLOCK_IP']));
+    ligne('SECURITY_AUTHENTICATION_NB_ATTEMPT', $l->g(1489), 'input', array('VALUE' => $values['ivalue']['SECURITY_AUTHENTICATION_NB_ATTEMPT'], 'SIZE' => 1, 'MAXLENGTH' => 3, 'JAVASCRIPT' => $numeric), '', '', $sup1);
+    ligne('SECURITY_AUTHENTICATION_TIME_BLOCK', $l->g(1490), 'input', array('END' => $l->g(511), 'VALUE' => $values['ivalue']['SECURITY_AUTHENTICATION_TIME_BLOCK'], 'SIZE' => 1, 'JAVASCRIPT' => $numeric), '', '', $sup1);
```
Comment (charleneauger) [inline require/function_config_generale.php:1024]: Bug -> possibility to write letters

**Pull request merged**